prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/NM/NM-receiver-transform.yml

```diff
@@ -0,0 +1,49 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  MSH: '/MSH'
+
+elements:
+  - name: nm-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
```

Comment:
This STLT has this setting enabled `"useBlankInsteadOfUnknown": "patient_race",`. We need to move that into a receiver transform. You need to check if patient race equals `Asked but unknown` or `Unknown` and set it to blank if it is. We need to check what Simple Report puts in the FHIR bundle when they send `Unkown` or `Asked but unknown` to make sure we're checking for the right values.

Reply:
Got it. How do I ask SR about this?

Reply:
You should be able to send a report through their UI with there should be an option to set the Patient's race to Unknown, or refused to answer. Then we can get the FHIR bundles for those and see how they send it to us.